Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "ChainRules"
 uuid = "082447d4-558c-5d27-93f4-14fc19e9eca2"
-version = "0.5.1"
+version = "0.5.2"
```

Comment:
this is that rare change that don't need a release, since it is only changing tests